Assets/Scripts/Combat/CombatController.cs

```diff
@@ -71,13 +85,40 @@ private void AttachTile(int moveCostOverride, Tile adjacentTile, Tile parent)
         }
     }
 
-    protected void FindSelectableTiles()
+    // Returns true if valid charge tiles were found.
+    protected bool FindSelectableChargeTiles()
+    {
+        ClearVisitedTiles();
```

Comment:
I would incorporate this as 1st line in the FindSelectableTiles method and get rid of it from here.

Reply:
Done